src/app/components/ScheduleControlPanelForm.tsx

```diff
@@ -104,6 +114,13 @@ export default class ScheduleControlPanelForm extends
                         message={<p style={{ fontSize: 14 }}>Please confirm that you would like to <b>permanently delete</b> the selected shift(s).</p>}
                         title="Delete Shift(s)"
                     />
+                    <Button 
+                        className="cancel-button" 
```

Comment:
cancel-button?  perhaps we need a better name for this if is used for more than one purpose:

`className='toolbar-button cancel'`   and `className='toolbar-button select-all'`